official/resnet/resnet_run_loop.py

```diff
@@ -158,7 +158,9 @@ def learning_rate_with_decay(
   initial_learning_rate = 0.1 * batch_size / batch_denom
   batches_per_epoch = num_images / batch_size
 
-  # Multiply the learning rate by 0.1 at 100, 150, and 200 epochs.
+  # Reduce the learning rate at certain epochs.
+  # CIFAR10:  divide by 10 at epoch 100, 150, and 200
```

Comment:
nit: CIFAR-10, also this make the colon align with each other.

Reply:
"also this make the colon align with each other." A man after my own heart.